docs/website/adr/007-standardize-log-output.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: 7
+title: |
+  7. Standardize log output
+authors:
+- name: Mithril Team
+tags: [Accepted]
+date: 2024-04-07
+---
+
+## Status
+
+Accepted
+
+## Context
+
+* Logs from Mithril client CLI are redirected to stderr. Only the result of the command execution is in `stdout`. This makes it possible to exploit the result, see our [blog post](/dev-blog/2024/02/26/mithril-client-cli-output-breaking-change).
+* Mithril aggregator logs are always redirected to `stdout` which is not consistent with the Mithril client CLI log strategy.
+
+## Decision
+
+* For commands that provide a result or execute an action, logs are sent to `stderr`. Only the result of the command is sent to `stdout`.
+* For commands that launch a program without an expected result (server), logs are sent to `stdout`.
+
+## Consequences
+
+* Need to use `stderr` to get all information logs.
```

Comment:
Maybe specify that end users relying on this would have a breaking change and would need to handle transition

Reply:
Done ✅